packages/react-hooks/src/useAccountInfo.ts

```diff
@@ -38,6 +41,8 @@ export function useAccountInfo (value: string | null, isContract = false): UseAc
   const { isAddress } = useAddresses();
   const accountInfo = useCall<DeriveAccountInfo>(api.derive.accounts.info as any, [value]);
   const accountFlags = useCall<DeriveAccountFlags>(api.derive.accounts.flags as any, [value]);
+  const nominator = useCall<Nominations>(api.query.staking.nominators as any, [value]);
+  const validator = useCall<ValidatorPrefs>(api.query.staking.validators as any, [value]);
```

Comment:
```suggestion
  const nominator = useCall<Nominations>(api.query.staking.nominator, [value]);
  const validator = useCall<ValidatorPrefs>(api.query.staking.validators, [value]);
```

The `as any` in the previous lines should be removed as well (at some point in the past the derives didn't work well/at-all typing-wise with `useCall`, so those were hacks. The `query.*` was always good :)